ui/guest/src/components/Guest.tsx

```diff
@@ -211,7 +228,7 @@ export function Guest(props) {
 
             (type === 'html') && plugins.push('quickbars');
 
-            // eslint-disable-next-line no-undef
+            // @ts-ignore
             if (!tinymce) {
```

Comment:
- change to `window.tinymce`
- declare above so the `@ts-ignore` comment is not required for tinymce usages